src/components/app/ProfileViewer.css

```diff
@@ -11,4 +11,14 @@
   background-color: #fff;
   animation-name: profileViewerFadeIn;
   animation-duration: 200ms;
-}
\ No newline at end of file
+}
+
+@media (max-width: 767px) {
+  .profileViewer {
+    width: 100%;
+    height: 100%;
+  }
+  body {
+    width: 100%;
+  }
+}
```

Comment:
I don't think the profile viewer is useful on mobile. I'm mostly interested by the Home. Can you please remove this part?